docs/xgcpolicy.md

```diff
@@ -127,24 +133,53 @@ The following options are ignored when specified with `-Xgcpolicy:balanced`:
 - `-Xmr<size>`
 - `-Xmrx<size>`
 - `-Xnoloa`
-- `-Xnopartialcompactgc` (deprecated)
-- `-Xpartialcompactgc` (deprecated)
+<!-- - `-Xnopartialcompactgc` (deprecated) -->
+<!-- - `-Xpartialcompactgc` (deprecated) -->
+
+
+### `optavgpause`
+
+        -Xgcpolicy:optavgpause
+
+: The "optimize for pause time" policy uses concurrent mark and concurrent sweep phases. Pause times are shorter than with `optthruput`, but application throughput is reduced because some garbage collection work is taking place in the context of mutator (application) threads, and GC frequency is increased.
```

Comment:
just English thing... should we repeat here 'because' (first reason is so long, that a reader might lose context when start reading about the second reason)?

and _because_ GC frequency is increased.